sktime/forecasting/statsforecast.py

```
@@ -552,3 +555,255 @@ def get_test_params(cls, parameter_set="default"):
+
+    The MSTL (Multiple Seasonal-Trend decomposition using LOESS) decomposes the time
+    series in multiple seasonalities using LOESS. Then forecasts the trend using
+    a custom non-seaonal model and each seasonality using a SeasonalNaive model.
+
+    Parameters
+    ----------
+    season_length : Union[int, List[int]]
+        Number of observations per unit of time. For multiple seasonalities use a list.
+    trend_forecaster : estimator, optional, default=AutoETS()
+
+
+    References
+    ----------
+    .. [1]
+        https://nixtla.github.io/statsforecast/src/core/models.html#multiple-seasonalities
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.statsforecast import StatsForecastMSTL
+
+    >>> y = load_airline()
+    >>> model = StatsForecastMSTL(season_length=[3,12])
+    >>> fitted_model = model.fit(y=y)
+    >>> y_pred = fitted_model.predict(fh=[1,2,3])
+    >>> y_pred
+    1961-01    454.507079
+    1961-02    419.341442
+    1961-03    477.373918
+    Freq: M, Name: Number of airline passengers, dtype: float64
+    """
+
+    _tags = {
+        "ignores-exogeneous-X": True,
+        "capability:pred_int": True,
+        "capability:pred_int:insample": True,
+    }
+
+    from sktime.forecasting.ets import AutoETS
+
+    def __init__(
+        self,
+        season_length: Union[int, List[int]],
+        trend_forecaster=AutoETS(),
+    ):
+        self.season_length = season_length
+        self.trend_forecaster = trend_forecaster
+
+        super().__init__()
+
+    def _instantiate_model(self):
+        """Create underlying forecaster instance."""
+        from statsforecast.models import MSTL
+
+        from sktime.forecasting.base._base import BaseForecaster
+
+        if issubclass(type(self.trend_forecaster), BaseForecaster):
+            self.trend_forecaster = StatsForecastBackAdapterMSTL(self.trend_forecaster)
+
```

What happens if the check fails? Is `MSTL` capable of handling any forecaster from any library?

If not, it should raise an error, or at least throw an warning.

---

Gotcha, done